docs/reference/ml/anomaly-detection/ml-configuring-alerts.asciidoc

```diff
@@ -291,7 +291,7 @@ The state of the job. It can be `opening`, `opened`, `closing`,
 
 ==== _Model memory limit reached_
 
-`context.message`::
+`context.message` ^*^::
 A preconstructed message for the rule.
 
 `context.results`::
```

Comment:
1. All fields from `context.rusults` are available. 
2. `memory_status` is `ok` for recovered alerts.